libs/ui/src/lib/record-thumbnail/record-thumbnail.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="h-full h-full relative flex-shrink-0 overflow-hidden">
+  <svg
+    class="text-gray-200 absolute h-12 w-12 transform -translate-x-1/2 -translate-y-1/2"
+    style="top: 50%; left: 50%;"
+    xmlns="http://www.w3.org/2000/svg"
+    fill="none"
+    viewBox="0 0 24 24"
+    stroke="currentColor"
+  >
+    <path
+      stroke-linecap="round"
+      stroke-linejoin="round"
+      stroke-width="2"
+      d="M4 16l4.586-4.586a2 2 0 012.828 0L16 16m-2-2l1.586-1.586a2 2 0 012.828 0L20 14m-6-6h.01M6 20h12a2 2 0 002-2V6a2 2 0 00-2-2H6a2 2 0 00-2 2v12a2 2 0 002 2z"
+    />
+  </svg>
+  <img
+    *ngIf="thumbnailUrl"
+    class="relative h-full w-full object-cover object-left-top"
+    [src]="thumbnailUrl"
+    alt="thumbnail"
```

Comment:
Add a translation

Reply:
in fact thumbnail in the model should have a label and be 0..n - so I'll wait for refactoring this before adding label which may be useless after.